book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+  end
+
+  def add_rental(_person, date)
+    Rental.new(date, book, self)
```

Comment:
- [ ] You probably wanted to write _person which you used for the parameter and not book, kindly fix it up.